src/index.js

```diff
@@ -306,7 +305,7 @@ class SimpleThermostat extends LitElement {
             `
           })}
         </section>
-        ${this.renderModeSelector(activeMode)}
+        ${this.renderModeSelector(state)}
```

Comment:
The problem with this is that you can set `mode_type: fan` to control fan modes with the card. It's annoying that the climate fixes removed the parity between `operation_mode` and `fan_mode` (and potentially other ones).